src/db.js

```diff
@@ -75,21 +95,37 @@ async function deleteIp(app, ip) {
  * @param {String} address the bb-promster address
  * @returns {Promise<IPutResponse>}
  */
-async function addApp(name, address) {
+export async function addApp(name, address) {
     let keyExists = await etcd.getAll().prefix(`${SERVICE_BASE_URL}/${name}/${address}`).keys();
     if (!keyExists || (Array.isArray(keyExists) && keyExists.length === 0)) {
         return etcd.put(`${SERVICE_BASE_URL}/${name}/${address}`).value("").exec();
     }
     throw Error("Duplicated app")
 }
 
+/**
+ * Adds a new app to be watch by Big Brother
+ * @param {String} name the service name
+ * @param {String} address the bb-promster address
+ * @returns {Promise<IPutResponse>}
+ */
+export async function addDescApp(name, desc) {
+  let keyExists = await etcd.getAll().prefix(`${DESC_BASE_URL}/${name}/${desc}`).keys();
+  if (!keyExists || (Array.isArray(keyExists) && keyExists.length === 0)) {
+      return etcd.put(`${DESC_BASE_URL}/${name}/${desc}`).value("").exec();
+  }
+  throw Error("Duplicated app")
```

Comment:
@gutorc92, it looks to me that this function only adds the description for an app, not the app. Is the comment `Adds a new app to be watch by Big Brother` wrong?